src/jobs/base.go

```diff
@@ -25,36 +25,51 @@ package jobs
 
 import (
 	"context"
+	"flag"
 	"time"
 
+	"github.com/google/uuid"
 	"go.uber.org/zap"
 
-	"github.com/Arriven/db1000n/src/utils/templates"
+	"github.com/Arriven/db1000n/src/utils"
 )
 
 // Args comment for linter
 type Args = map[string]interface{}
 
-// GlobalConfig is a struct meant to pass commandline arguments to every job
+// GlobalConfig passes commandline arguments to every job.
 type GlobalConfig struct {
+	ClientID string
+
 	ProxyURLs           string
-	ScaleFactor         int
 	SkipEncrypted       bool
-	Debug               bool
-	ClientID            string
 	EnablePrimitiveJobs bool
+	ScaleFactor         int
+
+	Debug bool
```

Comment:
I feel like this one is no longer used in jobs since I've switched debug logs to zap but it can be addressed in a separate PR